lib/tbot/service_spiffe_svid_output.go

```diff
@@ -50,61 +50,124 @@ const (
 
 // SPIFFESVIDOutputService
```

Comment:
It would be great if you could amend this doc comment. Knowing what kind of "output" we mean here, what is the service's role in the grand scheme of things, would be greatly appreciated and would help understanding it at glance.

Reply:
cd5fea0816